lib/taskbook.js

```diff
@@ -192,6 +250,62 @@ class Taskbook {
     return data;
   }
 
+  _filterOverdue(data) {
+    Object.entries(data)
+      .forEach(([id, item]) => {
+        if (!item.dueDate) {
+          return delete data[id];
+        }
+
+        const now = new Date();
+        const dueDate = parseDate(item.dueDate);
+        const isOverdue = isBefore(dueDate, now);
+
+        if (item.isComplete || !isOverdue) {
+          delete data[id];
+        }
+      });
+
+    return data;
+  }
+
+  _filterToday(data) {
```

Comment:
same as https://github.com/klauscfhq/taskbook/pull/69/files#r212086721

Reply:
For me it's a case of the clarity gained. Under the hood, `date-fns` is doing very simple date manipulations using the native `Date` type. I'm not sure why we'd ditch a well tested library in favour of just doing the same implementation within Taskbook?